features/polypolyMembership/src/context/membership-context.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import i18n from "!silly-i18n";
+
+import { useNavigate, useLocation } from "react-router-dom";
+
+export const MembershipContext = React.createContext();
+
+function updatePodNavigation(pod, handleBack, location) {
+    pod.polyNav.actions = {
+        back: () => handleBack(),
+    };
+    location.pathname !== "/onboarding" && location.pathname !== "/"
```

Comment:
Maybe even just `location.pathname !== "/"`? Don't exactly know how the logic works here

Reply:
Yes, I'm thinking we don't even need `"/onboarding"` since we're not navigating to that screen unless we press "back", but not using that route.